nusight2/src/client/components/robot_selector/robot_label/style.css

```
@@ -61,7 +61,7 @@
 
 .dataTable {
   display: grid;
-  grid-template-columns: 1fr 1fr;
+  grid-template-columns: 220px 180px;
```

```suggestion
  grid-template-columns: 180px 180px;
```

Pairs with the suggestion below.